profiler/shakesapp/go.mod

```diff
@@ -0,0 +1,12 @@
+module shakesapp
+
+go 1.14
+
+require (
+	cloud.google.com/go v0.52.0
+	cloud.google.com/go/storage v1.5.0
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
Looks like you need to run `go mod tidy`.

Reply:
Thanks! done.